src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -1066,6 +1056,10 @@ class PersonAgent(
   }
 
   val myUnhandled: StateFunction = {
+    case Event(BeamAgentSchedulerTimer, _) =>
+      // Put a breakpoitn here to see an internal state of the actor
+      log.info(s"Received message from ${sender()}")
```

Comment:
Should this remain or be put behind a config at least?

Reply:
switched it to a log.debug instead